apps/client/src/components/ConclusionPrint.js

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from "react";
+import { Paragraph, Heading, Icon } from "@datapunt/asc-ui";
+import { Alert } from "@datapunt/asc-assets";
+import uniqBy from "lodash.uniqby";
+
+import Context from "../context";
+import {
+  QuestionWrapper,
+  MainWrapper,
+  Question,
+  UserAnswer,
+  UserResult,
+  UserResultParagraph,
+} from "../pages/ResultsPageStyles";
+import { booleanOptions } from "./Question";
+
+export default ({ checker }) => {
+  const permitsPerQuestion = [];
+  const context = useContext(Context);
+  const {
+    streetName,
+    houseNumberFull,
+    postalCode,
+    residence,
+  } = context.address;
+
+  const event = new Date();
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+  };
+  const today = event.toLocaleDateString("nl-NL", options);
```

Comment:
This includes the time right? Can we name it `now` or `currentDateTime` or something?

Reply:
Done